### PR TITLE
Pylint configuration

### PR DESCRIPTION
**Pull request by Tichau**

### CODE REVIEW

**Comment by nadege** [comment]
It seems that you let all the defaults values beside the one about docstrings. Is it because it's the best choice?

File: pylintrc [hunk 1]
```diff
@@ -112,7 +112,7 @@ additional-builtins=
 
 # List of strings which can identify a callback function by name. A callback
 # name must start or end with one of those strings.
-callbacks=cb_,_cb
+callbacks=on_
```
Comment (Tichau) [inline pylintrc]: suggestion: I like to call callback onStuffHappen but I have no particular problems with stuffHappenCb naming. As you want.

File: pylintrc [hunk 2]
```diff
@@ -122,17 +122,17 @@ redefining-builtins-modules=six.moves,future.builtins
 [BASIC]
 
 # Good variable names which should always be accepted, separated by a comma
-good-names=i,j,k,ex,Run,_
+good-names=
```
Comment (Tichau) [inline pylintrc]: I don't like abreviation :) as you know

Comment (nadege) [inline pylintrc]: Me neither :) But keep the `_` please, because of http://stackoverflow.com/a/5893946/1359020

Comment (Tichau) [inline pylintrc]: Ok, i didn't understood the explanations but I trust you ^^

**Pull request merged**